### PR TITLE
[WIP] – Add section with ng add command to install tailwind into Angular project

### PR DESCRIPTION
**Pull request by raisiqueira**
Add section to Angular guide to install tailwind with `ng add` command to optimize developer time 😄.

### CODE REVIEW

**Comment by simonswiss** [comment]
Hey!

I have fixed the merge conflicts in this branch. For now I have put your copy (slightly edited) at the top of the file, but we need to figure out _where_ this copy would fit best, it's there as a placeholder for now.

Adding [WIP] to the PR title until we land on a solution that can be merged.

We're doing some thinking around places where there is a "fork in the road" in the guide at some point, where there are two possible decisions one could make. That's the case here, with using the `ng add` command vs doing it manually.

**Comment by raisiqueira** [comment]
Hi, @simonswiss. I think that at the top is the best posistion for `ng add` command.

File: src/pages/docs/guides/angular.mdx [hunk 1]
```diff
@@ -10,6 +10,21 @@ Launch day test: Angular works with the normal tailwindcss package 🎉
...
 
+## Automated add Tailwind in your Angular project with `ng add` command:
+
+You can save time by adding tailwind automatically. After creating a Angular project running the `ng new my-project` command, in your project folder, run the following command:
+
+```bash
+ng add @ngneat/tailwind
+```
+
+This command will be install `tailwindcss`, `postcss`, `autoprefixer`, `postcss-loader` and `@angular-builders/custom-webpack`. Will also create `webpack.config.js` and `tailwind.config.js` files.
```
Comment (vltansky) [inline src/pages/docs/guides/angular.mdx:21]: `@ngneat/tailwind` no longer installing all those plugins, but only `tailwindcss`, `@angular-builders/custom-webpack` and `postcss` (will be removed soon as Angular already support PostCSS 8)

**Review by vltansky — CHANGES_REQUESTED** [review]

**Comment by adamwathan** [comment]
Hey thanks for this! This is a bit too much work for us to review and fine-tune on our usual GitHub clean up schedule and needs to be schedule into our normal project calendar to properly process, so it may be several weeks before we get a chance to address this but leaving it open as we do intend to work through this one.

**Comment by raisiqueira** [comment]
Hi, @adamhopkinson. No problem, i undestand the situation.

**Comment by adamhopkinson** [comment]
I think the above ping should have been to @adamwathan, not me!

**Comment by adamwathan** [comment]
Hey thanks for this! We've removed the Angular guide from the repo since it was never properly finished/published, so unfortunately these changes no longer have a home. Hopefully have a chance to do a proper Angular guide in the future. I think some of this stuff has changed too since Angular added more official support for Tailwind recently?

Either way would definitely be open to a PR adding a guide for installing with Angular using the most up to date instructions, and following the conventions from our other guides. We'll get to making one ourselves eventually I hope but if someone else was able to put one together that matched the existing ones in structure and tone that would be great too.

**Pull request closed without merging**